tests/test_tension.py

```diff
@@ -0,0 +1,109 @@
+from anesthetic.examples.perfect_ns import correlated_gaussian
+import numpy as np
+from numpy.linalg import inv, solve, slogdet
+from anesthetic.tension import stats
+
+
+def test_tension_stats_compatiable_gaussian():
+    d = 3
+    V = 6.0
+    nlive = 1000
+    bounds = [[-1, 1], [0, 3], [0, 1]]
+
+    meanA = [0.1, 0.3, 0.5]
+    covA = np.array([[.01, 0.009, 0],
```

Comment:
Missing 0 in 0.01 (there are a few of these, I'm technically on holiday and using my phone so you can find the rest)

Reply:
done